src/main/java/uk/gov/hmcts/darts/addaudio/validator/AddAudioValidator.java

```diff
@@ -72,7 +88,38 @@ private void validateSize() {
         }
     }
 
+    private void validateDuration(Audio audio) {
+
+        OffsetDateTime startDate = OffsetDateTime.of(
```

Comment:
there is a MapperUtility.java that does this conversion for you, probbaly best to use that here and for finishDate

Reply:
refactored into MapperUtility.java